wagtail/api/v2/filters.py

```diff
@@ -73,33 +73,38 @@ def filter_queryset(self, request, queryset, view):
         Eg: ?order=random
         """
         if "order" in request.GET:
-            order_by = request.GET["order"]
+            order_by_list = request.GET["order"].split(',')
 
             # Random ordering
-            if order_by == "random":
+            if "random" in order_by_list:
+                if len(order_by_list) > 1:
+                    raise BadRequestError("random ordering cannot be combined with other fields")
```

Comment:
We should add a unit test for this additional case, so that it is clear that it's expected behaviour and does not break in the future.

Reply:
Looks like a test has been added for this. Thank you.